make_prg/subcommands/update.py

```diff
@@ -144,7 +144,7 @@ def run(cl_options):
     options = cl_options
     global update_shared_data
 
-    if io_utils.output_files_already_exist(options.output_prefix):
+    if io_utils.output_files_already_exist(options.force, options.output_type, options.output_prefix):
```

Comment:
Sorry to be a pain, but I think the simplest way to do this would have been 
```suggestion
    if io_utils.output_files_already_exist(options.output_type, options.output_prefix) and not options.force:
```
it doesn't really make sense to have force as a parameter to the function

Reply:
I do agree, fixed in https://github.com/iqbal-lab-org/make_prg/pull/35/commits/5d81c9f41d444f5e7bb5b86cd5caf91766147d60 , but I've put the `not options.force` condition first for very slightly better performance. Please check, thanks!